bootstrap/local/variables.tf

```
@@ -0,0 +1,97 @@
+variable "starter_module" {
+  description = "The starter module to use for the deployment. (e.g. 'basic')|1|starter_module"
+  type        = string
+  default     = "basic"
+}
+
+variable "target_directory" {
+  description = "The target directory to create the landing zone files in. (e.g. 'c:\\landingzones\\my_landing_zone')|2"
+  type        = string
+  default     = ""
+}
+
+variable "create_seed_resources_in_azure" {
```

```suggestion
variable "create_bootstrap_resources_in_azure" {
```
Just trying to be consistent, we seem to refer to these resources as accelerator resources or bootstrap resources